### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by aldemeery**
### Added

- Basic responsive layout of the page.
- Header styles across all screens.
- Navbar styles across all screens.
- Left aside styles across all screens.
- Main section styles across all screens.
- Magazine styles across all screens.
- Editor's Pick styles across all screens.
- Subscribe section styles across all screens.
- Footer styles across all screens.
- Right aside styles across all screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,6 +4,10 @@
         <meta charset="UTF-8" />
```
Comment (lucasmazo32) [inline index.html:4]: - [ ] Kindly fix the W3C validator errors.

> Screenshot
image

File: index.html [hunk 2]
```diff
@@ -14,7 +18,2098 @@
...
+                                    </div>
+                                    <div class="form-group">
+                                        <a
+                                            href="#"
+                                            class="btn btn-block btn-primary subscribe-cta"
+                                            >Sign up</a
+                                        >
+                                    </div>
+                                </form>
+                            </div>
```
Comment (lucasmazo32) [inline index.html:1041]: - [ ] Kindly change the space it can take for tablet view.

> Original
image

> Your website
image

File: index.html [hunk 3]
```diff
@@ -14,7 +18,2098 @@
...
+                </div>
+                <hr class="hr" />
+                <div class="news-card card unbordered col-12 col-md-3">
+                    <div class="card-img-container">
+                        <img
+                            class="card-img-top unbordered"
+                            src="https://dummyimage.com/286x180/aaa/fff.png&text=In+The+Magazine"
+                            alt="Card image cap"
+                        />
+                        <p class="category">Ipsum</p>
```
Comment (lucasmazo32) [inline index.html:1062]: - [ ] There are 8 articles or news in the original webpage. Please add the necessary ones to your page.

> Original
image

> Your page
image

File: index.html [hunk 4]
```diff
@@ -14,7 +18,2098 @@
...
+                            src="https://dummyimage.com/286x180/aaa/fff.png&text=Editor's+Pick"
+                            alt="Card image cap"
+                        />
+                        <p class="category">Ipsum</p>
+                    </div>
+                    <div class="card-body">
+                        <h5 class="card-title">
+                            <a href="#">
+                                Lorem ipsum dolor sit amet consecr adipisicing
+                                elit. Aliquam?
```
Comment (lucasmazo32) [inline index.html:1196]: - [ ] The section "Editor's pick" have two articles per row. Kindly change that.

> Original
image

> Your page
image

File: index.html [hunk 5]
```diff
@@ -14,7 +18,2098 @@
...
+            <!-- Footer -->
+            <div class="row">
+                <footer class="col-12 footer row">
+                    <div class="col-12 footer-top">
+                        <div class="footer-top-left">
+                            <img
+                                src="./assets/images/logo-red.svg"
+                                alt="logo"
+                                class="footer-logo"
+                            />
```
Comment (lucasmazo32) [inline index.html:2080]: - [ ] For some reason this logo doesn't show up in my screen. Can you check if that happens in yours?

> Logo
image

**Comment by aldemeery** [comment]
Issues Fixed.

**Pull request closed without merging**